Dockerfile

```diff
@@ -14,11 +14,16 @@
 
 FROM openjdk:8-alpine
 
-ENV SONATYPE_DIR=/opt/sonatype
 ENV SONATYPE_LIB=/opt/sonatype/lib
 ENV SONATYPE_BIN=/opt/sonatype/bin
 
+# symlink to shell script will go here
+ENV LIFECYCLE_BIN=/lifecycle
```

Comment:
wondering if we should just call this directory 'nexus' since that spans all of our product line?